pkg/sqlstore/impl/system/store.go

```diff
@@ -246,7 +246,7 @@ func (s *SystemStore) GetSchemaByTableName(ctx context.Context, name string) (sq
 		return sqlstore.TableSchema{}, fmt.Errorf("failed to get the table: %s", err)
 	}
 
-	index := strings.LastIndex(createStmt, "strict")
+	index := strings.LastIndex(strings.ToLower(createStmt), "strict")
```

Comment:
@brunocalza, when deploying to testnet I noticed that this caused a panic.
After some digging, it looks like for the older tables it returned `STRICT` (uppercase).

Probably this is related that in our previous parser we had some upper cases than we changed in the latest versions. And, probably SQLite stores in `sqlite_schema` the original statement executed. 

So this means that we'll have both cases in `sqlite_schema`:
1- All the existing tables before this parser upgrade that used uppercase
2- The new tables will use lowercase.

This is my theory of the explanation to this panic.

As you see, the fix is quite easy. But it's interesting to note that we should remember this about everything we do in `sqlite_schema` from now on, since this "uppercase/lowercase" situation can backfire again in the future. Probably the best way to deal with this is always remember to lowercase (or uppercase) before trying to do any "unstructured finding" (like `LastIndex` or similar).